packages/server/src/automations/steps/queryRows.js

```diff
@@ -127,9 +146,21 @@ exports.run = async function ({ inputs, appId }) {
     version: "1",
   })
   try {
-    await rowController.search(ctx)
+    let rows
+
+    if (
+      inputs.onEmptyFilter === EmptyFilterOptions.RETURN_NONE &&
+      inputs["filters-def"] &&
+      hasNullFilters(inputs["filters-def"])
+    ) {
+      rows = []
+    } else {
+      await rowController.search(ctx)
+      rows = ctx.body ? ctx.body.rows : []
```

Comment:
could this be `ctx.body?.rows || []`

Reply:
Unfortunately we can't use `body?` on the backend due to our node version